tests/system/source/main.d

```diff
@@ -80,6 +85,7 @@ int main (string[] args)
 
         checkBlockHeight(addresses, 1);
     }
+    // Warning: GenesisBlock() is invalid here
```

Comment:
Could you explain the reason why the GenesisBlock is invalid in this line, 88?

Reply:
Sure! I expanded the description.